File: 30-Days-of-Code/Day-3-Intro-to-Conditional-Statements/R/solution.r
f <- file("stdin")
open(f)
row <- readLines(f, n=1, warn = FALSE)
num <- row
close(f)

num = as.integer(num[1])

if (num %% 2 == 1) {
    cat("Weird")
} else if (num >= 6 && num <= 20) {
    cat("Weird")
} else
    cat("Not Weird")

File: 30-Days-of-Code/Day-3-Intro-to-Conditional-Statements/Rust/solution.rs
use std::io::{self,BufRead};

fn main() {
    let mut line = String::new();
    let stdin = io::stdin();
    
    line.clear(); 
    stdin.lock().read_line(&mut line).unwrap();
    let num = line.trim().parse::<u32>().unwrap();

    if num % 2 == 1 {
        println!("Weird");
    } else if num >= 6 && num <= 20 {
        println!("Weird");
    } else {
        println!("Not Weird");
    }
    
}
